### PR TITLE
Adn/env vars

### PR DESCRIPTION
**Pull request by alejandrodnm**

### CODE REVIEW

File: projects/pgai/pgai/cli.py [hunk 1]
```diff
@@ -117,13 +117,22 @@ def convert(self, value, param, ctx) -> int:  # type: ignore
...
+    return logging.getLevelName("INFO")  # type: ignore
+
+
 @click.command(name="worker")
 @click.version_option(version=__version__)
 @click.option(
     "-d",
     "--db-url",
     type=click.STRING,
-    envvar="VECTORIZER_DB_URL",
```
Comment (jgpruitt) [inline projects/pgai/pgai/cli.py:126]: The docs need to be updated to reflect the change in env var name.
https://github.com/timescale/pgai/blob/vectorizer/docs/self-hosting-vectorizer.md?plain=1#L42

**Review by jgpruitt — CHANGES_REQUESTED** [review]
one small thing

**Review by thedodd — APPROVED** [review]

File: projects/pgai/pgai/cli.py (hunk 1 above)
Comment (alejandrodnm) [reply projects/pgai/pgai/cli.py:126]: That's part of @billy-the-fish PR

**Review by Askir — APPROVED** [review]
Ayy thanks for fixing this, we can revert it once we drop support for 3.10 I suppose.

File: projects/pgai/pgai/cli.py (hunk 1 above)
Comment (alejandrodnm) [reply projects/pgai/pgai/cli.py:126]: https://github.com/timescale/pgai/pull/158#discussion_r1813163360

Comment (jgpruitt) [reply projects/pgai/pgai/cli.py:126]: ahh. ok

**Review by jgpruitt — APPROVED** [review]

**Pull request merged**